lib/src/screens/settings/widgets/settings_choices_cell.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:cake_wallet/view_model/settings/choices_list_item.dart';
+import 'package:flutter/material.dart';
+
+class SettingsChoicesCell extends StatelessWidget {
+  const SettingsChoicesCell(this.choicesListItem, {Key key}) : super(key: key);
+
+  final ChoicesListItem choicesListItem;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: Theme.of(context).backgroundColor,
+      padding: EdgeInsets.all(24),
+      child: Column(
+        mainAxisSize: MainAxisSize.min,
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Row(
+            children: [
+              Text(
+                choicesListItem.title,
+                style: TextStyle(
+                  fontSize: 14,
+                  fontWeight: FontWeight.normal,
+                  color: Theme.of(context).primaryTextTheme.title.color,
+                ),
+              ),
+            ],
+          ),
+          const SizedBox(height: 24),
+          Center(
+            child: SingleChildScrollView(
+              scrollDirection: Axis.horizontal,
+              child: Container(
+                decoration: BoxDecoration(
+                  borderRadius: BorderRadius.circular(30),
+                  color: Color(0xffF2F0FA),
+                ),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: choicesListItem.items.map((dynamic e) {
+                    final bool isSelected = choicesListItem.selectedItem == e;
```

Comment:
remove type definition for `final`

Reply:
why? it's more readable this way

Reply:
It's final and value will be not changed and definition type will not be changed in runtime too, as it's clean from right side expression that it's bool.